src/entities/course.entity.ts

```diff
@@ -0,0 +1,27 @@
+import { STORYBLOK_STORY_STATUS_ENUM } from '../utils/constants';
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity } from './base.entity';
+
+@Entity({ name: 'course' })
+export class CourseEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid', { name: 'courseId' })
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column()
+  slug: string;
+
+  @Column({
+    unique: true,
+    nullable: true,
+  })
+  status: STORYBLOK_STORY_STATUS_ENUM;
+
+  @Column({
+    unique: true,
+    nullable: true,
+  })
+  storyblokid: string;
```

Comment:
can we search and replace `storyblokid` with `storyblokId` for this PR and the following ones 🙏